Documentation/Plugins/Index.rst

```diff
@@ -283,13 +283,14 @@ The collection plugin shows one collection, all collections or selected collecti
  - :Property:
        targetPid
    :Data Type:
-       :ref:`t3tsref:data-type-page-id`
+       `t3tsref:data-type-page-id`
+   :
```

Comment:
This looks weird. Is this a typo or intended?

Reply:
Currently, unresolved references are not displayed. I have already opened an issue for this at https://github.com/TYPO3-Documentation/render-guides/issues/580. Although we could set a default value, doing so would require changing the test level of the documentation workflow from "Fail on Warning" to "Fail on Error." Therefore, I have converted these references into plain text. These references are also unresolved in the current documentation.

Reply:
My question was about the `:` in line 287. :)